.pylintrc

```diff
@@ -1,7 +1,7 @@
 [MASTER]
 
 # Disable the message, report, category or checker with the given id(s).
-disable=too-few-public-methods
+disable=too-few-public-methods,trailing-newlines
```

Comment:
What's the reasoning here? Seems like an easy thing to prevent.

Reply:
Oh, its not because its a problem. Im not sure if text files should not end with new lines!
hhttps://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline

Apparently, the POSIX standard requires there to be one!

Reply:
PEP8 (and pylint) allows there to be 1 trailing newline, but not multiple.

At least, I didn't get errors on files I created that have only 1 newline. But if it's acting up elsewhere, I'm okay to disable this.

Reply:
huh, my pylint constantly shows me warnings. In that case I can disable it if its behaving sanely for you guys